exercises/custom-converter/practice/src/failure.ts

```diff
@@ -0,0 +1,372 @@
+import {
```

Comment:
Wait!!! You should duplicate the code of `DefaultFailureConverter` here.

All you need to do is import that class from the SDK, and instantiate it with your custom options.

```
import { DefaultFailureConverter } from '@temporalio/common';

export const failureConverter = new DefaultFailureConverter({ encodeCommonAttributes: true });
```

Reply:
Nice!! Thanks for pointing that out!